data/resources/settings.json

```diff
@@ -28,6 +28,8 @@
     "User Interface": {
         "BuildMenu Position": "BOTTOM",
         "Language": "en",
-        "Font Filename": "resources/fonts/arcadeclassics.ttf"
+        "Font Filename": "resources/fonts/arcadeclassics.ttf",
+        "SubMenuButtonWidth": 32,
```

Comment:
This really shouldn't be in settings.json. I see the need for defining a common button width since not all images are 32 x 32. For now, let's leave it there since there's an ongoing discussion about where to go with the UI....

Reply:
Yeah, this is not the ideal location for this but I'd rather not hardcode it so I had to put it somewhere and there isn't anywhere in the UILayout.json to it, at the moment.